Jenkinsfile

```diff
@@ -198,21 +187,6 @@ EOF
 				  onlyStable: false,
 				  sourceEncoding: 'ASCII'
 			junit 'coverage/jest/junit.xml'
-			script {
-				catchError {
-					if(readFile(".cypress_status").trim() == '0'){
-					    status = 'SUCCESS'
```

Comment:
Don't we need to keep this to set the Jenkins build status based on the outcome of cypress?

Reply:
I think t's only for Cypress check status. We now don't need it. See updated PR desc